package.json

```diff
@@ -23,6 +23,8 @@
     "@release-it-plugins/workspaces": "4.2.0",
     "@release-it/conventional-changelog": "5.1.1",
     "conventional-recommended-bump": "7.0.1",
+    "prettier": "^3.3.2",
+    "pretty-quick": "^4.0.0",
```

Comment:
looks like husky or simple-git-hooks are needed for it to run as a commit hook: https://github.com/prettier/pretty-quick?tab=readme-ov-file#pre-commit-hook